styles/_forms.scss

```diff
@@ -374,48 +375,17 @@
 }
 
 .gc-file-input-upload-button {
-  @apply bg-gray-200 border-1 border-gray-400 text-black-default font-bold py-2 px-6 rounded-lg mb-14 text-center inline-block;
-  @apply md:w-full md:text-small_base cursor-pointer border-b-4 py-3 px-5 mr-2 m-0 text-base no-underline outline-none relative;
-
-  &:focus-within {
-    box-shadow: 0 0 0 2px white;
-    outline: solid 2px #303fc3;
-    outline-offset: 2px;
-    @apply bg-blue-focus;
-    @apply text-gray-50;
-  }
-
-  &:active {
-    box-shadow: 0 0 0 2px white;
-    outline: solid 2px #1b2736;
-    outline-offset: 2px;
-  }
-  &:hover {
-    @apply bg-gray-300;
-    @apply text-black-default;
-  }
-}
-
-/**
- * Don't want to use the 'standard' method
- * for visually hiding the native file upload
- * with CSS, because setting the form control
- * to 1px height/width will cause VoiceOver to
- * inaccurately announce the state of the
- * form control (e.g. VO will not announce
- * whether there are files selected or not.)
- */
-.gc-file-input [type="file"] {
-  font-size: 0.1px;
-  left: 0;
-  top: 0;
-  @apply absolute opacity-0 w-full h-full;
+  @apply text-center inline-block mb-0;
 }
 
 .gc-file-input-file-selected {
   @apply inline-block;
 }
 
+.gc-file-input [type="file"] {
```

Comment:
any reason why we have a seperate .gc-file-input selector to add `display: none`?

Reply:
The goal of this is to hide the default HTML file input button. Here is what happens without it ;)
<img width="844" alt="Screen Shot 2022-04-27 at 11 05 02 AM" src="https://user-images.githubusercontent.com/1763961/165549980-28d612c8-75d3-4fa5-9b3d-10a35fc25626.png">